### ISSUE TITLE
Cant change map width since new patch.

### ISSUE THREAD

**Issue by EveZeroEight** [exodus4d/pathfinder#935]
Ever since 2.0 dropped I can no longer change the width of the map to match the width of my screen. Now Im forced to have a module like routes or killmails next to the map. Is this intentional or can we still return it back to the way it used to be? Cheers.

**Comment by CalMatar** [comment]
No, The map width is now static. You can only make it taller. It is "wider" in the map but you can not make it wider on the screen with the added column on the side, which you can not remove. I would option for a layout without that but that is just me. I prefer to have all of that below the map and have my map full width the same as you but that is not my call.

**Comment by EveZeroEight** [comment]
Damn, this new layout really sucks. It messes with my OCD something fierce. It would be nice to be able to make the map same width of my screen again and have all the modules underneath like I used to :(

**Comment by exodus4d** [comment]
@CalMatar is right. For Screens <= 1600px you still get the "old" layout. So if you could life with that, just resize your browser. 
![resize](https://user-images.githubusercontent.com/9299033/78649798-3d8d2100-78be-11ea-9011-902cc4b12a5b.gif)

\>= 1600px you get the left/right column where you can drag into all modules (except the Signature panel) you want to. Very large screens will benefit from the new layout, because the total max-with (map width + left column)  is no longer limited:
![image](https://user-images.githubusercontent.com/9299033/78649939-7af1ae80-78be-11ea-95b3-3e4b0bd0444e.png)
 But yeah adding the "full page map view" (like before) for all breakpoints would be easy but it is currently not planned

Link: #470

**Comment by exodus4d** [comment]
The "Layout Settings"  dialog  got some new features:
- Toggle between "2 column"  and "3 column"  grid layout
- Toggle for "column reverse"  order

![layout_switch](https://user-images.githubusercontent.com/9299033/79377398-e6411d80-7f5b-11ea-9216-975dff959955.gif)
